compiler/optimizer/OptimizationUtil.hpp

```diff
@@ -25,10 +25,20 @@
 namespace TR
 {
 
-class OptimizationUtil: public TR::Allocatable<OptimizationUtil, TR::Allocator>
+class OptimizationUtil
 	{
 	public:
 
+   static void *operator new(size_t size, TR::Allocator a)
```

Comment:
Please fix the indentation in this file to make it consistent with the surrounding code.